### PR TITLE
Ethereum 1.3.1 - for hardhat deployer update

### PR DESCRIPTION
**Pull request by hellwolf**
Update CHANGELOG.md

### CODE REVIEW

**Comment by hellwolf** [comment]
@JoshuaTrujillo15

**Review by jtriley2p — APPROVED** [review]
LGTM

File: packages/ethereum-contracts/CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,26 @@ All notable changes to the ethereum-contracts will be documented in this file.
...
+### Added
+
+- BNB Chain Support (#925)
+- Initialize logic contracts on deployment (using "castrate") (#841)
+
+### Fixed
+
+- Queue up FAILED_VERIFICATIONS (#900)
+- Addresses several shortcomings in current worflows (mainly mainnet related) (#911)
+- Prod deploytment script fix #971
```
Comment (0xdavinchee) [inline packages/ethereum-contracts/CHANGELOG.md]: small typo, deploytment -> deployment

Comment (hellwolf) [reply packages/ethereum-contracts/CHANGELOG.md]: fixed

**Review by kasparkallas — APPROVED** [review]

**Review by 0xdavinchee — APPROVED** [review]

**Pull request merged**